tedana/resources/decision_trees/kundu.json

```diff
@@ -325,8 +363,11 @@
                     "unclass_highvar"
                 ]
             },
-            "kwargs": {},
-            "_comment": "Add more here"
+            "kwargs": {
+                "log_extra_info": "",
+                "log_extra_report": "If any components are still labeled as unclassified or unclassified high-variance, then a revised decision table score is calculated from the provisionally accepted, unclassified, and unclassified high-variance components."
+            },
+            "_comment": "Calculate a new d_table_score. Why can't we use the original d_table_score function? Because it's a metric function?"
```

Comment:
Is this a comment for us?

Reply:
Yeah the `_comment`s are, I believe, for developers.